internal/ent/schema/emailverificationtoken.go

```diff
@@ -0,0 +1,70 @@
+package schema
+
+import (
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/dialect/entsql"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+
+	"github.com/datumforge/datum/internal/ent/mixin"
+)
+
+// EmailVerificationToken holds the schema definition for the EmailVerificationToken entity
+type EmailVerificationToken struct {
+	ent.Schema
+}
+
+// Fields of the EmailVerificationToken
+func (EmailVerificationToken) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("token").
+			Comment("the verification token sent to the user via email which should only be provided to the /verify endpoint + handler").
+			Unique(),
+		field.Time("ttl").
+			Comment("the ttl of the verification token which needs to be explicitly set, otherwise defaults to the current time").
```

Comment:
```suggestion
			Comment("the ttl of the verification token which needs to be explicitly set").
```

This no longer sets a default, so this comment should be updated + schema regenerated.